src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetSystemInfo.cs

```diff
@@ -8,6 +8,6 @@ internal static partial class Interop
     internal static partial class Kernel32
     {
         [DllImport(Libraries.Kernel32)]
-        internal static extern void GetSystemInfo(out SYSTEM_INFO lpSystemInfo);
+        internal static unsafe extern void GetSystemInfo(SYSTEM_INFO* lpSystemInfo);
```

Comment:
Is unsafe change necessary? (it is still using the old DllImport)

Reply:
The parameter was switched to be a pointer - `SYSTEM_INFO*`

Reply:
yes but since title is "Use GeneratedDllImport..". this change and changes due to this change looked unrelated.

Reply:
The general goal of `GeneratedDllImport` (and this feature branch) is to remove p/invoke overhead - whether that is through using the DllImport source generator to generate the code or using it to find places where we could easily make the p/invoke inlinable.